tox.ini

```diff
@@ -23,7 +23,7 @@ setenv =
 	IIB_TESTING=true
 pytest_command =
     pytest -vv \
-        --capture=sys\
+        --capture=sys \
```

Comment:
Nope, the tox tests were failing. One of the errors was fixed by this change. The other error is due to some version error which I am working on fixing. Let me create another PR to fix tests and remove it from this PR to avoid confusion.

Without the space the command gets created like this, pytest -vv --capture=sys--cov-config .coveragerc --cov=iib --cov-report term --cov-report xml --cov-report html